### PR TITLE
Canopy frame utilization and updated MIBs

### PR DESCRIPTION
**Pull request by barryodonovan**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

I've added frame utilization graphing and the updated MIBs to poll for it.
I think that's all for (me for) now on canopy_generic 
There are a lot more useful metrics but if there is going to be a big refactoring of wireless devices then these could be part of the mix.

### CODE REVIEW

**Comment by murrant** [comment]
@barryodonovan Actually, if you know what kind of stats you are looking for on Canopy devices, can you list them on the community post to help out with the refactor, because we are a lacking a lot of info in that area.

https://community.librenms.org/t/improve-wifi-collection-and-graphing-code/453

**Comment by barryodonovan** [comment]
Yup, will do. Just trying to catch up with getting LibreNMS into production.
I lost January to everything that went wrong over christmas when I took time off! ;-)

File: includes/polling/wireless/canopy-generic.inc.php [hunk 1]
```diff
@@ -290,4 +290,21 @@
         $graphs['canopy_generic_freq'] = true;
         unset($rrd_filename, $freq);
     }
+
+    $downlinkutilization = str_replace('"', "", snmp_get($device, "frUtlLowTotalDownlinkUtilization.0", "-Ovqn", "WHISP-APS-MIB"));
```
Comment (murrant) [inline includes/polling/wireless/canopy-generic.inc.php]: Do you need the str_replace here?  You should get nicely formatted data here and not need that. (and the line below)

**Comment by murrant** [comment]
Also, can you `chmod 644` the mib files you added.  Thanks.

File: includes/polling/wireless/canopy-generic.inc.php (hunk 1 above)
Comment (barryodonovan) [inline includes/polling/wireless/canopy-generic.inc.php]: yeah, just kept things the same as the existing code so as not to confuse someone who's looking at it later, The entire thing need to be rewritten to use a multiget, I was just trying to shoehorn my requirements in within the time window I had.

I'll see if I can find time to rewrite today.

**Pull request merged**